judo-jsl-fullstack-karaf-project-template-application/src/main/resources/rest/pom.xml.hbs

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>{{ lowerCase model.name }}{{ projectPostfix }}</artifactId>
+        <groupId>{{ groupId }}</groupId>
+        <version>${revision}</version>
+    </parent>
+
+    <artifactId>{{ lowerCase model.name }}{{ projectPostfix }}-rest</artifactId>
+    <name>{{ model.name }} Jakarta RESTful Web Services</name>
```

Comment:
There is no dependency / module under this one with the Jakarta namespace, only CXF. Is this name correct?